paasta_tools/check_mesos_resource_utilization.py

```diff
@@ -83,6 +85,11 @@ def check_thresholds(percent):
         over_threshold = True
     else:
         output += "OK: CPU usage is under %d%%. (Currently at %f%%)\n" % cpu_print_tuple
+    if current_disk >= percent:
```

Comment:
can you add blank lines between each if/else in here? These blocks kind of run into each other.